Fix build error when building against VTK9

See https://slicer.cdash.org/index.php?project=SlicerPreview&date=2021-03-10&filtercount=1&showfilters=1&field1=buildname&compare1=63&value1=CMFreg

```
D:\D\P\S-0-E-b\CMFreg\ApplyMatrix\ApplyMatrix.cxx(31,10): fatal error C1083: Cannot open include file: 'vtkPolyDataReader.h': No such file or directory [D:\D\P\S-0-E-b\CMFreg-build\ApplyMatrix\ApplyMatrixLib.vcxproj]
```

--- a/ApplyMatrix/CMakeLists.txt
+++ b/ApplyMatrix/CMakeLists.txt
@@ -5,6 +5,7 @@ set(MODULE_NAME ApplyMatrix)
 
 set(MODULE_TARGET_LIBRARIES
   ${ITK_LIBRARIES}
+  ${VTK_LIBRARIES}
   )
 
 #-----------------------------------------------------------------------------

--- a/Growing/CMakeLists.txt
+++ b/Growing/CMakeLists.txt
@@ -5,6 +5,7 @@ set(MODULE_NAME Growing)
 
 set(MODULE_TARGET_LIBRARIES
   ${ITK_LIBRARIES}
+  ${VTK_LIBRARIES}
   )
 
 #-----------------------------------------------------------------------------

--- a/LabelAddition/CMakeLists.txt
+++ b/LabelAddition/CMakeLists.txt
@@ -5,6 +5,7 @@ set(MODULE_NAME LabelAddition)
 
 set(MODULE_TARGET_LIBRARIES
   ${ITK_LIBRARIES}
+  ${VTK_LIBRARIES}
   )
 
 #-----------------------------------------------------------------------------

--- a/LabelExtraction/CMakeLists.txt
+++ b/LabelExtraction/CMakeLists.txt
@@ -5,6 +5,7 @@ set(MODULE_NAME LabelExtraction)
 
 set(MODULE_TARGET_LIBRARIES
   ${ITK_LIBRARIES}
+  ${VTK_LIBRARIES}
   )
 
 #-----------------------------------------------------------------------------

--- a/MaskCreation/CMakeLists.txt
+++ b/MaskCreation/CMakeLists.txt
@@ -5,6 +5,7 @@ set(MODULE_NAME MaskCreation)
 
 set(MODULE_TARGET_LIBRARIES
   ${ITK_LIBRARIES}
+  ${VTK_LIBRARIES}
   )
 
 #-----------------------------------------------------------------------------

--- a/NonGrowing/CMakeLists.txt
+++ b/NonGrowing/CMakeLists.txt
@@ -5,6 +5,7 @@ set(MODULE_NAME NonGrowing)
 
 set(MODULE_TARGET_LIBRARIES
   ${ITK_LIBRARIES}
+  ${VTK_LIBRARIES}
   )
 
 #-----------------------------------------------------------------------------

